src/GitHub.VisualStudio.UI/UI/Controls/InfoPanel.xaml.cs

```diff
@@ -82,12 +75,14 @@ public InfoPanel()
             DataContext = this;
             Icon = Octicon.info;
             IconColor = InfoColorBrush;
+        }
 
-            LinkCommand = new RelayCommand(x =>
-            {
-                if (!String.IsNullOrEmpty(x as string))
-                    Browser.OpenUrl(new Uri((string)x));
-            });
+        void OpenHyperlink(object sender, ExecutedRoutedEventArgs e)
+        {
+            var url = e.Parameter.ToString();
+
+            if (!string.IsNullOrEmpty(url))
+                Browser.OpenUrl(new Uri(url));
```

Comment:
We set the urls in the info panels, it's not from user input, so they should never be malformed. If they are and throw, we should catch that in testing (we are clicking on every infopanel that we have during testing to make sure nothing regressed, right? 😉)